### PR TITLE
Renaming variadic macros

### PR DESCRIPTION
**Pull request by MingweiSamuel**

### CODE REVIEW

**Review by jhellerstein — COMMENTED** [review]
Please modify the examples that use demux to use the new names. Ideally change the names everywhere?

**Comment by MingweiSamuel** [comment]
Yup WIP

File: hydroflow_lang/src/graph/ops/demux.rs [hunk 1]
```diff
@@ -15,19 +15,19 @@ use syn::{Expr, Pat};
...
 ///
 /// Takes the input stream and allows the user to determine what elemnt(s) to
 /// deliver to any number of output streams.
 ///
 /// > Note: Downstream operators may need explicit type annotations.
 ///
-/// > Note: Import the [`Pusherator`](https://hydro-project.github.io/hydroflow/doc/pusherator/trait.Pusherator.html)
-/// > trait to use the [`.give(...)` method](https://hydro-project.github.io/hydroflow/doc/pusherator/trait.Pusherator.html#tymethod.give).
+/// > Note: The [`Pusherator`](https://hydro-project.github.io/hydroflow/doc/pusherator/trait.Pusherator.html)
+/// > trait is automatically imported to enable the [`.give(...)` method](https://hydro-project.github.io/hydroflow/doc/pusherator/trait.Pusherator.html#tymethod.give).
```
Comment (MingweiSamuel) [inline hydroflow_lang/src/graph/ops/demux.rs:27]: Also a little update here for #285

File: variadics/src/lib.rs [hunk 2]
```diff
@@ -0,0 +1,299 @@
...
+///     /// A variadic list of `Debug` items.
+///     pub variadic<T> DebugList where T: std::fmt::Debug {}
+/// }
+///
+/// let x = &var_expr!(1, "hello", 5.6);
+/// let _: &dyn DebugList = x;
+/// println!("{:?}", x);
+/// ```
+#[macro_export]
+macro_rules! variadic_trait {
```
Comment (MingweiSamuel) [inline variadics/src/lib.rs:176]: This file is the only change worth reviewing at all. Code below this review comment is all old, pretty much

**Review by jhellerstein — APPROVED** [review]
LGTM!

**Pull request merged**